conveyal/analysis-ui
# Clear and load opportunity dataset statuses appropriately for each region

Load statuses along with OD data on page load which should ensure only seeing statuses for the current region. Also, wait for status to clear before reloading the page (which should fix the error found in #990).

Fixes #990

## How to test

1. Upload OD dataset
2. When complete, leave the status message uncleared
3. Switch to another region's OD page. Ensure no message is shown.
4. Switch back to original region's OD page. Ensure status message is shown.
